modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/build.gradle

```diff
@@ -36,9 +36,10 @@ dependencies {
 	compileInclude group: "com.liferay", name: "org.apache.logging.log4j", version: "2.17.1.LIFERAY-PATCHED-1"
 	compileInclude group: "com.liferay", name: "org.apache.logging.log4j.core", version: "2.17.1.LIFERAY-PATCHED-1"
 	compileInclude group: "com.liferay", name: "org.elasticsearch.plugin.legacy.geo", version: elasticsearchClientVersion
-	compileInclude group: "commons-io", name: "commons-io", version: "2.11.0"
+	compileInclude group: "commons-io", name: "commons-io", version: "2.15.1"
 	compileInclude group: "commons-lang", name: "commons-lang", version: "2.6"
 	compileInclude group: "org.apache.commons", name: "commons-compress", version: "1.26.0"
+	compileInclude group: "org.apache.commons", name: "commons-lang3", version: "3.14.0"
```

Comment:
Hi @holatuwol ,

As we have compileInclude `commons-lang3: 3.14.0` and `commons-compress: 1.26.0` in shared-dependencies-apache-commons, can we export the necessary packages in shared-dependencies-apache-commons, and use **compileOnly** in portal-search-elasticsearch7-impl to get these packages, please?

Thanks,
Jeff